variantlib/constants.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 
 VARIANT_HASH_LEN = 8
-CONFIG_FILENAME = "wheelvariant.toml"
+CONFIG_FILENAME = "variants.toml"
```

Comment:
Not sure about this change. In the end, it kinda assumes that we will always be dealing with wheel-oriented packages and nothing else would use the same filename. For example, rattler-build currently uses `variants.yaml` which is pretty close to colliding.

Reply:
FYI I searched the entire github - There is not one one file called `variants.toml` anywhere ...
Hence I simplified ;)

Reply:
Actually, thinking about it more, perhaps it should be named `variantlib.toml`, since it configures `variantlib`.

Reply:
let's keep `variants.toml` for now - very easy to change given that it's only in one place in the codebase